service/gcs/core/gcs/uvm.go

```diff
@@ -169,6 +169,55 @@ func (h *Host) CreateContainer(id string, settings *prot.VMHostedContainerSettin
 	// Clear the windows section of the config
 	settings.OCISpecification.Windows = nil
 
+	// Check if we need to do any capability/device mappings
+	if settings.OCISpecification.Linux.Resources != nil {
+		if settings.OCISpecification.Linux.Resources.Devices != nil {
+			// We use the device list access/allow to detect the privileged
+			// case. Check that here.
+			if len(settings.OCISpecification.Linux.Resources.Devices) == 1 {
+				if settings.OCISpecification.Linux.Resources.Devices[0].Allow == true ||
+					settings.OCISpecification.Linux.Resources.Devices[0].Access == "rwm" {
+
+					logrus.Debugf("opengcs::CreateContainer - spec.Linux.Resources.Devices set for privileged container")

```

Comment:
Not sure the logging protocol throughout this area of code, but should this be `WithFields`...?